app/views/middleware_server/_dr_reports_list.html.haml

```diff
@@ -0,0 +1,72 @@
+- if @record.middleware_diagnostic_reports.any?
+  .row#mw_dr_header
+    .col-md-12
+      %h3.pointer{:onclick => 'miqCallAngular({name: "toggleShowDiagnosticReports", args: []});'}
+        %span.fa.fa-angle-down
+        = _('JDR Reports')
+  .row#mw_dr_section
+    .col-md-12
+      .form-group
+        .dropdown.btn-group
+          %button#dr_download_dd.btn.btn-default.dropdown-toggle{'data-toggle' => 'dropdown'}
+            %span.fa.fa-download
+            %span.caret
+          %ul.dropdown-menu{:role => 'menu', 'aria-labelledby' => 'dr_download_dd'}
+            %li{:role => 'presentation'}
+              %a.pointer{:role     => 'menuitem',
+                         :tabindex => '-1',
+                         :onclick  => "DoNav('#{url_for(:action => 'dr_report_download', :id => to_cid(@record.id), 'download_type' => 'text')}');"}
+                %span.fa.fa-file-text-o
+                = _('Download as Text')
+            %li{:role => 'presentation'}
+              %a.pointer{:role     => 'menuitem',
+                         :tabindex => '-1',
+                         :onclick  => "DoNav('#{url_for(:action => 'dr_report_download', :id => to_cid(@record.id), 'download_type' => 'csv')}');"}
+                %span.fa.fa-file-text-o
+                = _('Download as CSV')
+        - if role_allows?(:feature => :middleware_dr_delete)
+          %button#dr_btn_delete.btn.btn-default{'onclick' => 'miqCallAngular({name: "deleteSelectedDr", args: []});', :disabled => 'disabled'}
```

Comment:
@israel-hdez it is recommended to have buttons in toolbars on the top, we usually do not recommend having buttons in middle of screen. cc @dclarizio @martinpovolny what do you guys think?

Reply:
same comment for download buttons.

Reply:
@h-kataria we've already have had some discussion about this. See https://github.com/ManageIQ/manageiq-ui-classic/pull/1764#issuecomment-319537310, https://github.com/ManageIQ/manageiq-ui-classic/pull/1764#issuecomment-319682183, https://github.com/ManageIQ/manageiq-ui-classic/pull/1764#issuecomment-320059528. It's a usability concern.

Reply:
@israel-hdez I am also a bit confused even after following the discussion. right now per your screenshot above, there is download next to each row and in addition also in the menu above the reports?

Reply:
@abonas The download button on each row is for download the JDR report. The one that is on top of the table is for downloading the table as CSV or Text, which is the same functionality as the download buttons available when you are browsing servers, datasources, etc.

Reply:
@israel-hdez it is confusing imo. perhaps some tooltip should refine this

Reply:
I added the tooltips: https://github.com/ManageIQ/manageiq-ui-classic/pull/1764/commits/b3637c5a6681215fee8a4286799df3cce146513e#diff-002598c1d95530eae2e85c719166439fR11 and: https://github.com/ManageIQ/manageiq-ui-classic/pull/1764/commits/b3637c5a6681215fee8a4286799df3cce146513e#diff-002598c1d95530eae2e85c719166439fR70

I hope it's ok.